src/player.cpp

```diff
@@ -3747,23 +3753,6 @@ bool Player::hasOutfit(uint32_t lookType, uint8_t addons)
 	return false;
 }
 
-bool Player::canLogout()
-{
-	if (isConnecting) {
-		return false;
-	}
-
-	if (getTile()->hasFlag(TILESTATE_NOLOGOUT)) {
-		return false;
-	}
-
-	if (getTile()->hasFlag(TILESTATE_PROTECTIONZONE)) {
-		return true;
-	}
-
-	return !isPzLocked() && !hasCondition(CONDITION_INFIGHT);
```

Comment:
I have not transfered the in-fight check, as this was keeping player online forever 🤷

I do not know if being pz-locked disabled the kick, @EPuncker?

Reply:
in pz locked situation the kick time is always 60 seconds

Reply:
alright, i'll update it to work like that later 💩